### PR TITLE
DlgSort

### PR DESCRIPTION
**Pull request by stevenndungu**
@africanmathsinitiative/developers  The bug fixed, someone to re-review when free. thanks

### CODE REVIEW

File: instat/dlgSort.Designer.vb [hunk 1]
```diff
@@ -75,6 +75,13 @@ Partial Class dlgSort
         Me.rdoAscending.Text = "Ascending"
         Me.rdoAscending.UseVisualStyleBackColor = True
         '
+        'ucrPnlOrder
+        '
+        Me.ucrPnlOrder.Location = New System.Drawing.Point(6, 15)
+        Me.ucrPnlOrder.Name = "ucrPnlOrder"
+        Me.ucrPnlOrder.Size = New System.Drawing.Size(127, 86)
+        Me.ucrPnlOrder.TabIndex = 0
```
Comment (Lunalo) [inline instat/dlgSort.Designer.vb]: Tab order need to be fixed

File: instat/dlgSort.vb [hunk 2]
```diff
@@ -74,16 +74,17 @@ Public Class dlgSort
...
 
         'Set radiobutton parameters
-        ucrPanelOrder.SetParameter(New RParameter("decreasing", 2))
-        ucrPanelOrder.AddRadioButton(rdoAscending, "FALSE")
-        ucrPanelOrder.AddRadioButton(rdoDescending, "TRUE")
-        ucrPanelOrder.SetRDefault("FALSE")
+        ucrPnlOrder.SetParameter(New RParameter("decreasing", 2))
+        ucrPnlOrder.AddRadioButton(rdoAscending, "FALSE")
+        ucrPnlOrder.AddRadioButton(rdoDescending, "TRUE")
+        ucrPnlOrder.SetRDefault("FALSE")
```
Comment (Lunalo) [inline instat/dlgSort.vb]: You need to confirm if R default is true for Ascending

File: instat/dlgSort.Designer.vb (hunk 1 above)
Comment (stevenndungu) [inline instat/dlgSort.Designer.vb]: @Lunalo Tab order is sorted, am not sure what exactly needs to fixed with it, perhaps check on the design(or design code) to confirm

File: instat/dlgSort.vb (hunk 2 above)
Comment (stevenndungu) [inline instat/dlgSort.vb]: from R `sort(x, decreasing = FALSE, ...)`

Comment (Lunalo) [inline instat/dlgSort.vb]: great thanks.

File: instat/dlgSort.Designer.vb (hunk 1 above)
Comment (Lunalo) [inline instat/dlgSort.Designer.vb]: Me.ucrPnlOrder.TabIndex = 0
This is what made me wonder if the order was done appropriately.
Thanks

**Comment by Lunalo** [comment]
@dannyparsons 

This is ready

**Comment by dannyparsons** [comment]
@stevenndungu Are you fixing layout issues now? I don't mind if you don't have time now, I can merge it.

**Comment by stevenndungu** [comment]
I made the changes in the last commit @lilyclements is this okay?

**Comment by lilyclements** [comment]
@dannyparsons This is ready for review

**Pull request merged**